pkg/jx/cmd/controller_workflow.go

```diff
@@ -507,6 +519,9 @@ func (o *ControllerWorkflowOptions) pollGitStatusforPipeline(activity *v1.Pipeli
 			log.Warnf("Failed to get PR number: %s", err)
 			return
 		}
+		if prNumber == 42 {
+			fmt.Printf("!!!Stop\n")
+		}
```

Comment:
wat?

Reply:
D'oh - this was from when I was testing out the fix. But it got merged before I had a chance to remove it! :-(
See https://github.com/jenkins-x/jx/pull/3053